### ISSUE TITLE
This is not an optimal way to share multiple projects. no .sln 

### ISSUE THREAD

**Issue by Daulaires** [cmooref17/ReservedItemSlotMods#37]
no way of really knowing how to implement this into other projects as it just isn't very clear. 
Consider making a read me for each one so we have some idea how to interact with this code.

**Comment by cmooref17** [comment]
At the moment, the mod page has some examples on how to use the main parts of the API if developers are wanting to reference any of it in their mod. I could get those examples added to this readme as well, but I'm heading to bed now, so it will have to be tomorrow. Is this what you were referring to, or were you more talking about the actual project/code instead of the mod?

Also, you're right about the no sln. I think the csproj is also not here, but I will look at this tomorrow or Monday when I'm free.

**Comment by Daulaires** [comment]
Yeah I am more talking about the actual project/code being used, and like with no .sln, or readme I feel a bit clueless as to how I can utilize just one thing instead of all the reserveditemdata slots, Like if I just wanted FlashlightReservedSlot it is a bit more difficult to understand how to just use that code. Maybe my reason isn't too clear, but basically just how to use certain bits and parts. If my reason isn't too clear, please do tell me, as I am sometimes not very clear on what I mean. 

Take your time, and mainly just wanted the readme cause my buddies just wanted the reservedFlashlight, and all the patches he wanted, but in one single mod, so he doesn't have to add multiple mods in.
